lib/r10k/git/cache.rb

```diff
@@ -16,7 +16,7 @@ class R10K::Git::Cache
 
   include R10K::Settings::Mixin
 
-  def_setting_attr :cache_root, File.expand_path(ENV['HOME'] ? '~/.r10k/git': '/root/.r10k/git')
+  def_setting_attr :cache_root, File.expand_path(ENV['HOME'] ? '~/.r10k/cache': '/root/.r10k/cache')
```

Comment:
I changed this because originally, there was only the git cache. Now, we're adding the forge module cache to it, and in my opinion, they should be using the same root.